lib/aiko/oled.py

```diff
@@ -140,9 +140,26 @@ def on_oled_message(topic, payload_in):
       oled.show()
     return True
 
+  # oled:text x y message
+  # If x or y are missing or are malformed (not integer), then we'll log what we can and move on
   if payload_in.startswith("(oled:text "):
-    tokens = payload_in[11:-1].split()
-    text = " ".join(tokens[2:])
+    try:
+      tokens = payload_in[11:-1].split()
+      if (len(tokens) < 3):
+        print("#### Missing parameters to oled:text. Format expected is (oled:text x y message)")
+        log(payload_in[11:-1])
```

Comment:
not related to your code per se, but my understanding is that print goes to the python console which you see if you are connected via usb.  
But where does log go?

Reply:
@geekscape 
and by that, I mean, it does go to the screen (which is a bit weird, I wouldn't call a screen print function "log"), but does it get logged anywhere else? MQTT?

Reply:
@marcmerlin [aiko.common.log(message)](https://github.com/geekscape/aiko_engine_mp/blob/master/lib/aiko/common.py#L34) is intended to provide a diagnostic log to whatever output options are available.  It is currently a work-in-progress ... and the [OLED screens set themselves up to be logging outputs](https://github.com/geekscape/aiko_engine_mp/blob/master/lib/aiko/oled.py#L104).
Since the OLED log() scrolls up existing text and writes to the bottom of the screen, it is effectively working like a console logging function, just the output goes to the OLED, rather than the USB serial terminal console.

Reply:
Got it, that makes sense.
So if I'm writing on the OLED to display output at a specific place that isn't meant to be a log, I won't be using the logging function.